### DISCUSSION TITLE
Communication

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#3093] [Ideas]
Possibly until next year, i have a lot of time on my hands, and i've found i'm spending more and more time here, probably to the chagrin of the maintainers who, i think, have less time to be here and find me to be a PIA.

Transmission has been my go-to client of choice for years, but i would only haphazardly use it, and i think that's key to what i'm going to propose next, because i've noticed there are users who are 24/7 users.

I repeat, there are people who run Transmission 24 hours a day and 7 days a week, and apart from some critical network infrastructure applications, i can't think of many applications users prioritise in such a way. Given how vociferous the users can get, 'It doesn't have this feature, i'm leaving' or 'This peeve has peeved me a lot, we're breaking up' i can't think of many applications that have that level of ire except for people's choice of browser, or possibly more applicable, OS. Because i've become so involved somehow, i'm now paying for a static ip because of a bug in Transmission, so i kinda get it. 

I digress. Often. And a lot.

Every other week on Github, or as i noticed last night in the forums, there is a post asking if Transmission is dead. Part of my intention for working on https://github.com/transmission/transmission/issues/2610 was to alleviate some of that which is why i wanted the new site to go live immediately with a prominent 'News' section. My background is digital communications, ux and ui design, web marketing and related 'stuff'.

So?

Even if it is in the interim until the website launches and 4.0 beta arrives, i want to set up a bi-annual newsletter. I don't need much time or help from the maintainers, i think i can handle most of it, especially with some help from other contributors, but the project is currently f&&(ing terrible with its external communications. I mean, professionally, if i were to grade the project's communications? It is worse than an F. To stretch this allegory, the dog ate our homework _and_ our grandmother died for the fifth time in two weeks, and we're somehow still unbothered that we will use the same excuse and she'll die again next week and hope that's acceptable.

Can we please improve this? And i'm happy to try and fix it. @ckerr @livings124 @mikedld

**Comment by ckerr** [comment]
I'm :+1: on the idea of having some better form of communication and would appreciate the help. I'm not so much in love with the idea of a bi-annual newsletter. I'd prefer to avoid making any commitments that follow a timeline; this project has had long periods of inactivity in the past, and we're all volunteers with day jobs here. So I'm very leery of making commitments that I don't know the project can deliver over a longer term.

(Also, the idea of a newsletter also begs the question of where that would be published and how people would find it?)

Possibly the best thing we could do would be more traction on #2610. It's not clear to me what the status of that issue is, it reads like this still needs php in order to work, which is a nonstarter on github pages, which is a static site? But that discussion probably should take place in #2610.

I agree with the problem though; I'm just not sure what the right solution is...?

**Reply by GaryElshaw** [reply]
Hi Charles, again, welcome back. I seem to be hearing a lot of not this, or not now, and i'm running out of suggestions unless the maintainers want to delve into a social media presence? There is a single aim here, and it's just that users know the project is alive. It has never been about assurances of any beta or release date. To be honest, i think users are happier knowing nothing is happening than assuming that nothing is happening - at all. 

If you re-read through what i was saying during the website development, we discussed the question of whether the forum in its present form should remain as is. The danger, as i advise all of my clients, is that if they don't have the human resources to manage their channels, it makes them look negligent because all the users see is themselves talking in a vacuum.

I was offering to build a newsletter using the usual online newsletter channels because i think it would be low volume anyway. But seeing this? Why not use it? 2011? https://twitter.com/transmissionbt

**Reply by ile6695** [reply]
> (Also, the idea of a newsletter also begs the question of where that would be published and how people would find it?)

We could use the Releases-page.

**Reply by GaryElshaw** [reply]
There are multiple avenues.

**Comment by GaryElshaw** [comment]
@ckerr @livings124 @mikedld If you want to give me the login to the Twitter account i'm happy to help manage it or at least fix up the visuals with the latest icon etc. Four suggested tweets.

1. Hello?
2. Is this thing on?
3. Contrary to unpopular opinion, we're not dead
4. We're actually very much alive https://github.com/transmission/transmission/commits/main

**Comment by Patrick010** [comment]
@GaryElshaw Good to hear that Transmission isnt dead, and on top of that, you mentioned a v4 beta :) Can you tell us a bit more on that? Maybe some sneak preview or some teaser screenshots? Would be great to know what to expect and what the new version will bring us.

**Reply by GaryElshaw** [reply]
The best sneak peak there is - a nightly build! https://transmissionbt.com/download/

**Reply by Patrick010** [reply]
Found it and got it installed. One question though, how do I change the language, because now I have localised Dutch version and the translation is mildly depressing ;) There's no language choice in the Settings menu.

**Reply by GaryElshaw** [reply]
What OS are you using @Patrick010? I'm on a Mac.

**Reply by Patrick010** [reply]
Win, Linux or Synology DSM

**Reply by GaryElshaw** [reply]
Sorry Patrick, i wouldn't know how to adjust language preferences on those OS's.

**Reply by Patrick010** [reply]
No problem. How do you do this on a Mac? Maybe its not all that different on a Windows of Linux machine.

**Reply by GaryElshaw** [reply]
Usually just altering the language preferences in the system prefs. Another option would be to build from source?

**Reply by Patrick010** [reply]
So Transmission selects language based on regional OS settings? Thats going to confuse a lot of people who want a different language. This should be selectable in Transmission application settings. Also, having Transmission and Application settings under 2 separate buttons doesnt make sense. One would expect just a Settings menu with all available options. 
@ckerr  @livings124 @mikedld is this something for you? Not sure who's developer or project member.

**Reply by GaryElshaw** [reply]
On MacOS it's normal for this to be handled at the OS level.

**Reply by Patrick010** [reply]
Yes, but what if you'd want to select a different gui language?

**Reply by GaryElshaw** [reply]
There's also the option of a per app basis too.
![SCR-20220523-vbf](https://user-images.githubusercontent.com/69029666/169801032-734ead29-b991-4ceb-afe6-4144ef4e6aff.png)
